linux_os/guide/services/ssh/ssh_server/firewalld_sshd_port_enabled/rule.yml

```diff
@@ -28,11 +28,30 @@ references:
     nist: AC-17(a),CM-6(b),CM-7(a),CM-7(b)
     srg: SRG-OS-000096-GPOS-00050
 
-ocil_clause: 'sshd service is disabled by firewall'
+platform: machine
+
+ocil_clause: 'sshd service is not enabled in the proper firewalld zone'
+
 ocil: |
     {{{ ocil_firewalld_allow_access(port="22", proto="tcp", service="ssh") }}}
 
 fixtext: |-
-    Enable sshd in firewalld configuration.
+    Enable SSH service in firewalld configuration.
 
     {{{ describe_firewalld_allow(proto="tcp", service="ssh") }}}
+
+warnings:
+    - general: |-
+        The remediation for this rule uses <tt>firewall-cmd</tt> and <tt>nmcli</tt> tools.
+        Therefore, it will only be executed if <tt>firewalld</tt> and <tt>NetworkManager</tt>
+        services are running. Otherwise, the remediation will be aborted and a informative message
+        will be shown in the remediation report.
+        These respective services will not be started in order to preserve any intentional change
+        in network components related to firewall and network interfaces.
+    - general: |-
+        This rule also checks if the SSH port was modified by the administrator and is reflecting
+        the expected port number. Although this is checked, fixing the custom ssh.xml file is not
```

Comment:
I think people can be confused when they read about this "custom ssh.xml" file because it comes out of nowhere and isn't explained what it is.

Reply:
It is natural for who is used to `firewalld` and I mentioned in the previous sentence "modified by the administrator". Maybe I could use the absolute path to make it more clear.

Reply:
Done. Could you take a look now, please?

Reply:
It's great